operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -81,6 +81,10 @@ mimir:
   # -- Base config file for Grafana Mimir and Grafana Enterprise Metrics. Contains Helm templates that are evaulated at install/upgrade.
   # To modify the resulting configuration, either copy and alter 'mimir.config' as a whole or use the 'mimir.structuredConfig' to add and modify certain YAML elements.
   config: |
+    usage_stats:
+      enabled: true
```

Comment:
Similar to colega's feedback, I think we can omit this field since it is the default.